otx/api/configuration/helper/convert.py

```diff
@@ -131,7 +131,7 @@ def convert(
     if target == str:
         result = yaml.dump(config_dict)
     elif target == dict:
-        result = config_dict
+        result = config_dict  # type: ignore
```

Comment:
Can we clarify type of config_dict?

Reply:
Yes, but problem is a type of `result`. Because `result` is set as string at line 132, mypy makes an error that result type (str) is not fit to config_dict. Actually, I don't know why mypy makes an error now although it has been ok before this PR and I didn't modify relevant things.